src/style.css

```diff
@@ -1,3 +1,61 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+:root {
+  --active-color: rgba(0, 172, 220, 1);
+  --main-color: rgba(0, 0, 0, 1);
+}
+
 body {
   margin: 0;
+  box-sizing: border-box;
+}
+
+html {
+  min-width: 1024px;
+  font-family: "Roboto", sans-serif;
+}
+
+.nav {
+  position: relative;
+}
+
+.nav__link {
+  font-size: 12px;
+  color: var(--main-color);
+  text-decoration: none;
+  cursor: pointer;
+  display: block;
+  margin-right: 20px;
+  text-transform: uppercase;
+}
+
+.nav__list {
+  list-style: none;
+  display: flex;
+  padding: 0;
+
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  margin: 10px 50px;
```

Comment:
padding fits better in this case (you want to add space inside a header, not move it from other elements on the page)
but vertical is redundant at all here - you shouldn't set it for margin. instead of doing it set the correct padding for links

Reply:
use padding: 0 50px;